movies/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "An example of a Hydra API, that allows to manage movie resources.",
   "main": "src/index.js",
   "scripts": {
-    "test": "jest",
+    "hypertest-compile": "hypertest-compiler src/tests",
+    "test": "jest; npm run test:analyse; npm run test:e2e",
+    "test:analyse": "hydra-validator analyse http://localhost:3000",
```

Comment:
Few words on what's going on would be welcomed, i.e. there are two analysis and validator invocations

Reply:
Um, I think you're not parsing it right. Let me break it up for you. The npm test script runs three commands:

* `jest`
* `npm run test:analyse`
* `npm run test:e2e`

The latter two are, respectively:

1. static analysis (same as https://analyse.hypermedia.app)
2. e2e tests defined in the `tests/tests.hydra` file

Additionally, right before the e2e run, the `.hydra` file is transformed into json for the runner to consume. In the long run I'd like this to be unnecessary but it's what it is right now

Reply:
Maybe it's worth of adding it to the readme somewhere?